docs/semantics-of-k.tex

```diff
@@ -3884,7 +3884,6 @@ \subsection{Matching Logic Theories}
 \end{equation}
 checks whether a list of sorts are declared:
 \begin{align*}
-& \KsortsDeclared{\shs} \colon \KSortList \to \shs \\
```

Comment:
It seems like this is redundant but maybe I misunderstood it.

Reply:
It does seem redundant, indeed, so removing it seems like the right choice.
Since you're at it, though, maybe you could wrap the following usages into `\parametric`, to get their `{` `}` around the `\shs` argument.